src/Command/Generate.php

```diff
@@ -61,44 +64,24 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 				$rootDir = getcwd();
```

Comment:
there's another '/' on L59, but github won't let me comment on it because it wasn't changed

Reply:
FFS! possibly more in the handlers

Reply:
yeah almost certainly, but i figure that can be another PR

Reply:
it's a different repo, so it'd have to be 😜

Reply:
I totally forgot about that